xtra-libp2p-offer/src/taker.rs

```diff
@@ -25,10 +26,11 @@ impl Actor {
 
         let task = async move {
             let offers = protocol::recv(stream).await?;
-
-            tracing::debug!(%peer, ?offers, "Received offers");
-
-            maker_offers.send(LatestMakerOffers(offers)).await?;
+            let span = tracing::debug_span!("Received offers", %peer, ?offers);
```

Comment:
nit: perhaps more readable name: 

```suggestion
            let span = tracing::debug_span!("Received new offers from maker", %peer, ?offers);
```